### ISSUE TITLE
This will likely fail on large commits with lots of file changes.

### ISSUE THREAD

**Issue by TheSeanLavery** [TheSeanLavery/code-review-gpt#15]
The bot should maybe give some prompt response about its too much to parse, but offer to parse certain files if they are important.

**Comment by CodeReviewGPT** [comment]
That's a great suggestion! We could have the bot give a prompt response about it being too much to parse, and then offer to parse certain files if they are important. Do you have any examples of what types of files you think should be parsed? Additionally, do you have any ideas of what the prompt response should look like? We can use that feedback to start crafting the code to make this happen.

**Comment by TheSeanLavery** [comment]
We could split the code up into smallter parts and have it loop through reviewing the code seperately, then taking the multiple outputs and have it combine all the feedback into one cohesive comment. Do you have any other suggestions to parse all the date?

**Comment by CodeReviewGPT** [comment]
Yes, another suggestion could be to use a regex script that looks for certain patterns in the code. This could be useful for reviewing large chunks of code quickly.

**Comment by TheSeanLavery** [comment]
Confirmed, there was a PR that was too large and it did not run and threw and error.

**Comment by CodeReviewGPT** [comment]
That's really unfortunate. Have you been able to determine what went wrong with the PR so we can avoid the same issue in the future?
